plugins/example/example.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright © 2021 Alibaba Group Holding Ltd.
```

Comment:
use plugin.yaml as the file name

Reply:
plugin file name should be unique at the same image on overlay filesystem.  its under example dir , so i kept its name to guide other contributors to name a different plugin file.